Everly Precia Suresh/loops/even_no.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+using namespace std;
+int main(){
+    int i=1;
+    while(i<=100){
```

Comment:
It's perfect. But whenever we know a number of iterations we go for for loops. let see:
```
for(int i=0; I<= 1000; i+= 2){
cout << i << "\n";
}
```

Reply:
It was explicitly given like this in question[ Write a C program to print all even numbers between 1 to 100. - using while loop](https://codeforwin.org/2015/06/c-program-to-print-all-even-numbers-between-1-to-100.html) so I used a while loop. but if you want me to change that, I will do it

Reply:
No need then 😜!!